### DISCUSSION TITLE
HTML Custom Panel

### DISCUSSION THREAD

**Discussion by minaspapadim** [pyscada/PyScada#165] [Q&A] [answered]
How can I use the variable values on a custom HTML panel? Do I need to install Websocket?

I tried this code but I don't get any value,
![image](https://github.com/user-attachments/assets/f6acd7e3-6690-4889-9631-636913ccf739)

**Comment by clavay** [comment]
You are right, I saw that recent changes made this code not working.
I will fix it and give you a solution next week.
I will put this in the documentation also.

**Reply by minaspapadim** [reply]
Any news about it?

**Reply by clavay** [reply]
Can you try : 
```
<div class="label" style="color: black;">TEST 
<span class="control-item type-numeric var-3519" style="color: black;">Loading</span>
</div>
```

Here the variable ID is 3519, replace it with your variable ID.

**Reply by minaspapadim** [reply]
It is not working either.

**Reply by clavay** [reply, marked as answer]
Can you try this branch : https://github.com/clavay/PyScada/tree/fixCustomHTMLPanel

**Reply by clavay** [reply]
Any news ?

**Reply by minaspapadim** [reply]
With your change and the code you mentioned above, it doesn't work.

**Reply by minaspapadim** [reply]
Sorry my bad, I forgot to use collectstatic after the update. It works as expected now. Thank you!
